src/Mollie/WC/Plugin.php

```diff
@@ -1103,5 +1103,21 @@ private static function extractRemoteItemsIds(array $data)
 
         return implode(',', wp_list_pluck($data['lines'], 'id'));
     }
+
+    /**
+     * @return array
+     */
+    public static function paymentMethodsImages()
+    {
+        $list = [];
+        $methods = getApiMethods();
+        if($methods){
+            $listIds = array_column($methods, 'id');
+            $list = array_combine($listIds, $methods);
+            return $list;
+        }else{
+            return $list;
+        }
```

Comment:
When you have something like `if return` you don't need to have `else` block, usually you don't need `else` at all.

I would rewrite it this way.

```
public static function paymentMethodsImages()
{
    $list = [];
    $methods = getApiMethods();

    if ($methods) {
        $listIds = array_column($methods, 'id');
        $list = array_combine($listIds, $methods);
    }
            
     return $list;
}
```

See how if there are methods you update the list with the values, otherwise the list will still be empty.